bot/config.py

```diff
@@ -401,6 +233,12 @@
     default_languages = safe_load(file)
 
 
+with open("bot/assets/reddit.json", "r") as f:
+    contents = f.read()
+
+reddit = json.loads(contents)
+
+
```

Comment:
There is no need to clutter `config.py` with this, you can do this directly in the `reddit` cog